shared/src/lib/sql/SqlGenerator.ts

```diff
@@ -1,13 +1,27 @@
 import { Dialect, KnexDialect, Schema, SchemaItem } from '../dialects/models'
 import {Knex} from 'knex'
 import knexlib from 'knex'
+import { BigQueryClient } from '../knex-bigquery'
+
+
+// eslint-disable-next-line
```

Comment:
I was running into this with Cassandra Knex being a bit of a pill, do we need the "require" stuff here? If we do, can you add a comment saying "please save this jic aaaah"

Reply:
Delete or uncomment.